main/src/com/google/refine/exporters/sql/SqlCreateBuilder.java

```diff
@@ -53,6 +53,10 @@ public SqlCreateBuilder(String table, List<String> columns, JsonNode sqlOptions)
 
     }
 
+    public static String addQuotes(String columnName) {
+        return "\"" + columnName + "\" ";
```

Comment:
It feels a bit arbitrary to have this method return an extra space at the end. If you want to introduce a dedicated method for this, I would argue it should:
* do the quoting properly (meaning that any quote in the column name would be escaped)
* not return a space at the end: the caller is responsible for adding it on their side if they are needed

Reply:
In the commit below (https://github.com/OpenRefine/OpenRefine/pull/5674/commits/54a76f0fc9a55fee7a840e16747130873692ffaf) : 

- I added escaping for double quotes. I didn't add escaping for single quotes because they're already working as expected (if I add escaping like this: `...replace("'", "''")` it exports two single quotes). 
- Removed space in the function and added it to each caller when needed